### PR TITLE
Add source KWU Landkreis Oder-Spree

### PR DESCRIPTION
**Pull request by maverickme**

### CODE REVIEW

File: custom_components/waste_collection_schedule/waste_collection_schedule/source/kwu_de.py [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+        r = requests.get("https://www.kwu-entsorgung.de/inc/wordpress/kal_objauswahl.php", headers=HEADERS)
+        parsed_html = BeautifulSoup(r.text, "html.parser")
+        Orte = parsed_html.find_all('option')
+
+        for Ort in Orte:
+            if self._city in Ort.text:
+                OrtValue = Ort['value']
+                break
+
+        r = requests.get("https://www.kwu-entsorgung.de/inc/wordpress/kal_str2ort.php?ort=" + OrtValue, headers=HEADERS)
```
Comment (mampfes) [inline custom_components/waste_collection_schedule/waste_collection_schedule/source/kwu_de.py:51]: Please use the params argument to pass parameters in URLs: https://requests.readthedocs.io/en/latest/user/quickstart/#passing-parameters-in-urls
This also ensures correct encoding.

**Review by mampfes — CHANGES_REQUESTED** [review]

File: custom_components/waste_collection_schedule/waste_collection_schedule/source/kwu_de.py (hunk 1 above)
Comment (maverickme) [reply custom_components/waste_collection_schedule/waste_collection_schedule/source/kwu_de.py]: Ok. Done. 👍

**Comment by maverickme** [comment]
.

**Comment by mampfes** [comment]
Sorry - I forgot this yesterday: Please add new source also to info.md

**Comment by mampfes** [comment]
Thanks for your contribution.

**Pull request merged**